static/js/pages/elections.js

```diff
@@ -55,28 +55,29 @@ var electionColumns = [
     }
   },
   {data: 'party_full', className: 'all'},
-  {
-    data: 'total_receipts',
-    orderSequence: ['desc', 'asc'],
-    render: tables.buildTotalLink(['receipts'], function(data, type, row, meta) {
-      return {
-        committee_id: row.committee_ids,
-        cycle: context.election.cycle
-      };
-    })
-  },
-  {
-    data: 'total_disbursements',
-    orderSequence: ['desc', 'asc'],
-    render: tables.buildTotalLink(['disbursements'], function(data, type, row, meta) {
-      return {
-        committee_id: row.committee_ids,
-        cycle: context.election.cycle
-      };
-    })
-  },
+  tables.currencyColumn({data: 'total_receipts', orderSequence: ['desc', 'asc']}),
+  tables.currencyColumn({data: 'total_disbursements', orderSequence: ['desc', 'asc']}),
   tables.barCurrencyColumn({data: 'cash_on_hand_end_period'}),
-  tables.urlColumn('pdf_url', {data: 'document_description', className: 'all', orderable: false})
+  {
+    render: function(data, type, row, meta) {
+      var dates = helpers.cycleDates(context.election.cycle);
+      var url = helpers.buildAppUrl(
+        ['filings'],
+        {
+          committee_id: row.committee_ids,
```

Comment:
Identifier 'committee_id' is not in camel case.<br>Identifier 'committee_ids' is not in camel case.